pennylane/transforms/optimization/merge_rotations.py

```diff
@@ -128,17 +127,25 @@ def qfunc(x, y, z):
             # If next gate is of the same type, we can merge the angles
             if current_gate.name == next_gate.name and current_gate.wires == next_gate.wires:
                 list_copy.pop(next_gate_idx + 1)
-
                 # The Rot gate must be treated separately
                 if current_gate.name == "Rot":
-                    cumulative_angles = fuse_rot_angles(
-                        cumulative_angles, cast_like(stack(next_gate.parameters), cumulative_angles)
-                    )
+                    if is_abstract(cumulative_angles):
+                        # jax-jit does not support cast_like
+                        cumulative_angles = cumulative_angles + stack(next_gate.parameters)
```

Comment:
This line doesn't seem to be covered as per Codecov, is that so?

Reply:
<img width="1114" alt="Capture d’écran 2022-05-05 à 15 38 32" src="https://user-images.githubusercontent.com/36766072/167012959-954a7aea-090c-4430-bb3f-9003b51f801f.png">